xdma-test.c

```diff
@@ -359,10 +360,16 @@ int main(int argc, char **argv)
 
 	if (!op && !reset) {
 		log_err("No operation specified, aborting.\n");
+		printf("%s", _help);
 		rc = -EINVAL;
 		goto done;
 	}
 
+	if (op && !host_addr) {
```

Comment:
host_addr == 0 is valid input though. You may need a flag like op or reset to determine whether the user has passed in the host address

Reply:
true. Let me fix it.